src/lib/pages/evm-contract-details/index.tsx

```diff
@@ -160,6 +162,12 @@ const EvmContractDetailsBody = ({
             <CustomTab onClick={handleTabChange(TabIndex.Transactions)}>
               Transactions
             </CustomTab>
+            <CustomTab
+              hidden={!evm.enabled}
+              onClick={handleTabChange(TabIndex.Holders)}
+            >
+              Holders
+            </CustomTab>
           </TabList>
```

Comment:
I think we don't need to have a hidden condition because we're on evm-contract-details.